solr/packaging/build.gradle

```diff
@@ -115,15 +121,15 @@ distributions {
         into "server"
       })
 
+      from(configurations.docs, {
+        into "docs"
+      })
       // docs/   - TODO: this is assembled via XSLT... leaving out for now.
+      // -- Is there more to do here?
```

Comment:
I don't think so. Documentation in Solr's Artifacts on Jenkins already contain everything perfectly. If its the same for Lucene, we're fine. Should land in the ZIP's `docs` folder.

Reply:
Oh, I am not sure if this works. At least in current master, I see NO 'docs/' foder in the binary releaese!

Reply:
See comments below.

Reply:
I think we can create a separate issue for collecting Solr docs? This patch is mostly about Lucene and it's pretty big already.